identity-service/src/app.js

```diff
@@ -187,6 +201,69 @@ class App {
     return audiusInstance
   }
 
+  async configureRewardsAttester(libs) {
+    // Make a more greppable child logger
+    const childLogger = logger.child({'service': 'RewardsAttester'})
+
+    // Await for optimizely config so we know 
+    // whether rewards attestation is enabled,
+    // returning early if false
+    await this.optimizelyPromise
+    const isEnabled = getFeatureFlag(this.optimizelyClientInstance, FEATURE_FLAGS.REWARDS_ATTESTATION_ENABLED)
+    if (!isEnabled) {
+      childLogger.info('Attestation disabled!')
+      return
+    }
+
+    // Fetch the challengeDenyList, used to filter out
```

Comment:
what is this exactly?

Reply:
It gives us the ability to selectively turn off arbitrary challenges that might easily exploitable. Will also need to piped into AAO. Can clarify comment